databases/mydb/models/history.go

```diff
@@ -73,121 +73,6 @@ var HistoryTableColumns = struct {
 
 // Generated where
 
-type whereHelperint64 struct{ field string }
```

Comment:
why ?
what changed in `History` model in this PR ?

Reply:
fixed